src/types.ts

```diff
@@ -176,5 +176,3 @@ export interface DoubleCsrfUtilities {
    */
   doubleCsrfProtection: doubleCsrfProtection;
 }
-
-export {};
```

Comment:
This isn't actually needed, without it, rollup builds things properly, so this avoids the issue that required the 3.0.5 and 3.0.6 releases